pom.xml

```diff
@@ -89,6 +89,11 @@
                 <artifactId>subversion</artifactId>
                 <version>2.5</version>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>copyartifact</artifactId>
+                <version>1.33-SNAPSHOT</version>
```

Comment:
The `version` can be placed directly in `aggregator/pom.xml`. The only reason for having `dependencyManagement` is when multiple modules need to use the same version of a dependency, which is not the case here.